src/index.html

```diff
@@ -6,8 +6,55 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Space+Grotesk:wght@700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__img">
+      </div>
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">
+        Product code: 195434
+      </div>
+      <div class="rating">
+        <div class="stars">
+          <div class="stars__star star-1">
+          </div>
+          <div class="stars__star star-2">
+          </div>
+          <div class="stars__star star-3">
+          </div>
+          <div class="stars__star star-4">
+          </div>
+          <div class="stars__star star-5">
+          </div>
+        </div>
+        <div class="reviews">
+          Reviews: 5
+        </div>
+      </div>
+      <div class="card__price">
```

Comment:
Add a blank line between the blocks
```suggestion
      </div>
      
      <div class="card__price">
```